app.js

```diff
@@ -18,6 +17,8 @@
   fallback.style.lineHeight = fallbackOutput.style.lineHeight = '28px';
   webfont.style.lineHeight = webfontOutput.style.lineHeight = '28px';
 
+  updateClipboardButtons();
```

Comment:
this has to be after the style assignments so copying works even if no adjustments are made